src/components/tutorial-collection-cards/components/card-heading/company-logo/index.tsx

```diff
@@ -1,28 +0,0 @@
-import Image from 'next/image'
```

Comment:
Pretty tangential to the PR, but during type work I moved `CompanyLogoOption` around, and that brought to my attention these unused sub-components of `card-heading`.

Happy to drop these from this PR & handle in a follow-up if we wanna do that instead 🙇